.buildkite/pipeline.yml

```diff
@@ -95,6 +95,10 @@ steps:
           queue: opensource
         plugins:
           artifacts#v1.9.3:
+            test-collector#v1.10.2:
+              files: "reports/TEST-*.xml"
+              format: "junit"
+              branch: "^main|next$$"
```

Comment:
Sorry, I didn't notice before that this was put in the middle of the `artifacts` plugin section.  I'm amazed that the pipeline even starts given how sensitive it is with me at times!